src/components/ProjectList.js

```diff
@@ -2,11 +2,21 @@ import React from "react";
 import ProjectItem from "./ProjectItem";
 
 function ProjectList({ projects }) {
-  console.log(projects);
   return (
     <div id="projects">
       <h2>My Projects</h2>
-      <div id="project-list">{/* render ProjectItem components here */}</div>
+      <div id="project-list">
+        {projects.map(project => {
+         return (
+                <ProjectItem 
+                  key={project.id} 
+                  name={project.name} 
+                  about={project.about} 
+                  technologies={project.technologies} 
+                />
+              )
+         })}
+      </div>
```

Comment:
Your spacing here is inconsistent which seems to be a recurring pattern. Please pay closer attention to it going forward.